integration-tests/tests/pages/search.page.ts

```diff
@@ -75,9 +78,14 @@ export class SearchPage {
         return this.page.locator('[data-testid="sequence-row"]');
     }
 
-    async clickOnSequence(rowIndex = 0) {
+    async clickOnSequenceAndGetAccession(rowIndex = 0): Promise<string | null> {
```

Comment:
I would keep the `clickOnSequence` as there a lot of tests that use it and don't use the accession

Reply:
haha ok I'll add a second function.

Reply:
done